elastalert/ruletypes.py

```diff
@@ -1288,6 +1288,9 @@ def check_matches(self, timestamp, query_key, aggregation_data):
                     if query_key is not None:
                         match = expand_string_into_dict(match, self.rules['query_key'], query_key)
                     self.add_match(match)
+                    percentage_format_string = self.rules.get('percentage_format_string', None)
+                    if percentage_format_string is not None:
+                        match['percentage'] = percentage_format_string % (match_percentage)
```

Comment:
Doing this could affect those who count on having access to the raw percentage value. What do you think about making this key `percentage_formatted` or something along those lines so it won't overwrite the raw `percentage` key?

Reply:
```suggestion
                        match['percentage_formatted'] = percentage_format_string % (match_percentage)
```